src/Service.Tests/SqlTests/GraphQLCollationTests/MsSqlGraphQLCollationTests.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Azure.DataApiBuilder.Service.Tests.SqlTests.GraphQLCollationTests
+{
+    /// <summary>
+    /// Test GraphQL Collations validating proper operations.
+    /// </summary>
+    [TestClass, TestCategory(TestCategory.MSSQL)]
+    public class MsSqlGraphQLCollationTests : GraphQLCollationTestBase
+    {
+        /// <summary>
+        /// Set the database engine for the tests
+        /// </summary>
+        [ClassInitialize]
+        public static async Task SetupAsync(TestContext context)
+        {
+            DatabaseEngine = TestCategory.MSSQL;
+            await InitializeTestFixture();
+        }
+
+        #region Tests
+        /// <summary>
+        /// MsSql Capitalization Collation Tests
```

Comment:
what do you mean by capitalization collation test here?
e.g. This test validates that DAB creates queries with consistent capitalization of database object references (Title, TITLE, title)
or 
This test validates that DAB creates queries with all lower case letters (Title, TITLE -> title)
or
This test validates that DAB creates queries with casing consistent with the requirements of the backend database: database X has case sensitive requirements vs not.